common/install_miopen.sh

```diff
@@ -34,15 +34,9 @@ retry () {
 
 # Build custom MIOpen to use comgr for offline compilation.
 
-## Need a sanitized ROCM_VERSION without patchlevel; patchlevel version 0 must be added to paths.
-ROCM_DOTS=$(echo ${ROCM_VERSION} | tr -d -c '.' | wc -c)
-if [[ ${ROCM_DOTS} == 1 ]]; then
-    ROCM_VERSION_NOPATCH="${ROCM_VERSION}"
-    ROCM_INSTALL_PATH="/opt/rocm-${ROCM_VERSION}.0"
-else
-    ROCM_VERSION_NOPATCH="${ROCM_VERSION%.*}"
-    ROCM_INSTALL_PATH="/opt/rocm-${ROCM_VERSION}"
-fi
+# Create /opt/rocm link if it doesn't exist
+ln -sf /opt/rocm-${ROCM_VERSION}* /opt/rocm
```

Comment:
@amd-aakash @WBobby This line is probably unnecessary, if all our rocm installations always install an `/opt/rocm` link. Would you say that is the case?

Reply:
"[ ! -L /opt/rocm ] && ln -sf /opt/rocm-${ROCM_VERSION}* /opt/rocm" may make it robust.